donate/payments/sqs.py

```diff
@@ -19,13 +23,25 @@ def sqs_client():
 
 
 def send_to_sqs(payload):
+    send_message_retries = 3
     # If BASKET_SQS_QUEUE_URL is not configured, do nothing (djangorq is logging the payload).
     if settings.BASKET_SQS_QUEUE_URL:
         client = sqs_client()
         if client is None:
+            logger.error("Could not connect to SQS Client.")
             return
 
-        return client.send_message(
-            QueueUrl=settings.BASKET_SQS_QUEUE_URL,
-            MessageBody=json.dumps(payload, cls=DjangoJSONEncoder, sort_keys=True),
-        )
+    for n in range(send_message_retries):
+        try:
+            client.send_message(
+                QueueUrl=settings.BASKET_SQS_QUEUE_URL,
+                MessageBody=json.dumps(payload, cls=DjangoJSONEncoder, sort_keys=True),
+            )
+
+            break
+
+        except botocore.exceptions.ClientError as err:
```

Comment:
Using the [boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/error-handling.html), I am listening for any error with the client while trying to use the "send_message" function, if there is one, the logger prints it out and will wait 5 seconds before trying a maximum of 3 times.